### PR TITLE
Editing

### PR DESCRIPTION
**Pull request by konradpab**
### Brief summary
[just one or few sentences, what is this blog post or change about]

### Reviewer checklist
* [ ] Post is a good read
* [ ] There are no grammar mistakes (run spellcheck)
* [ ] Proposed solution is good (can this be solved in another way? any possibly introduced bugs?)
* [ ] Do we list alternative approaches?
* [ ] Code looks good (is it easily understood? does it follow the [style guide](https://github.com/Appsilon/styleguide/blob/master/R-style-guide/R.md)?)
* [ ] Works as expected in test (check out and test, try to break it)
* [ ] Blog still works great on mobile (check out and test, try to break it)

### CODE REVIEW

File: _posts_knitr/2017-07-20-improve-retail.Rmd [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+![A high level diagram of supervised machine learning process](/assets/article_images/2017-07-20-improve-retail/supervised-machine-learning.png)
+
+Let’s get started and build some interesting applications of data science. First off, we need to agglomerate all of the relevant data we have on our clients. Certain regions have legal restrictions on what sensitive information you can store. There is a way around this; I am not advocating for breaking the law but rather, to anonymize your data. Many times this can be done with a common user ID across every system you use. I also want to be clear and point out that there are certain types of data that we simply can not collect. Always ask a lawyer about your options when dealing with sensitive information. 
+
+![Map of global data protection. It is a bit out of date but the overall message the the same](/assets/article_images/2017-07-20-improve-retail/privacy-map.jpg)
```
Comment (filipstachura) [inline _posts_knitr/2017-07-20-improve-retail.Rmd]: is the same

File: _posts_knitr/2017-07-20-improve-retail.Rmd [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+Flexibility is exactly what we need when doing data science; the entire structure of a data warehouse would need changing whenever we want to try something new. This may seem amusing, but is only the tip of what insights lie below the surface. Stream processing lets us react near instantaneously to changes in our client's behavior. For example, they walk into our store and purchase an item on our site at a lower rate. We lost money here because our dynamic pricing did not take into account the client's location. This may not always be possible but should get those gears spinning as to when else we can achieve.
+
+## Use Cases
+
+Retail and e-commerce both generate a surprisingly large amount of data in recent years. But without the proper tools in place, most of this data illustrates what you already know. I want to point out a few common uses of data science in retail. There are quite a few more that we could discuss, but not every company is ready for them. Contact us if you think you are ready for full automation!What I mean is that they are trying to skip steps that need to be done before we start completely automating processes.
```
Comment (filipstachura) [inline _posts_knitr/2017-07-20-improve-retail.Rmd]: Please remove "Contact us if you think you are ready for full automation!".

Instead, we can add at the end of the paragraph sth like: 'At Appsilon we help companies at every stage of data science maturity take the next step'.

File: _posts_knitr/2017-07-20-improve-retail.Rmd [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+### Product Recommendations
+
+Traditionally, retailers would have consultants in store that would use their own judgement to help customers find what they need. There are also encaps that present the highest bidder to customers without taking into consideration their interests or needs. Online, this would be based on the past revenue and would try and recommend the highest selling products. This didn’t really make any sense, as these products were doing quite well selling themselves and skewed the rest of the catalogue. 
+
+There are a few ways for recommendations to reach your clients. The first most basic example is where we do not know our client at all. They are literally a ghost and we don’t even know where they are browsing from. Without using data science we can recommend a high margin item or the most common purchase our clients make as the first thing our new visitor sees.
```
Comment (filipstachura) [inline _posts_knitr/2017-07-20-improve-retail.Rmd]: Your recommendations are based on intuition but are not correct. As an example let suppose we have 3 products:
A) margin: 10$ probability of purchase: 10% = expected margin is 1$
B) margin: 5$ probability of purchase: 30% = expected margin is 1.5$
C) margin: 1$ probability of purchase: 60%  = expected margin is 0.6%

The best product to recommend is B, even though it is not most common (less than C) and not with the highest margin (less than A).

Comment (filipstachura) [inline _posts_knitr/2017-07-20-improve-retail.Rmd]: We always can get some data:
- weekday, time of a day
- IP address -> geolocation
- browser data (mobile or desktop? internet browser?)
- behavioral data (how do they browse our website)

File: _posts_knitr/2017-07-20-improve-retail.Rmd [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+If we are diligent in our data science approach and streamline our inventory to a near optimal level, savings start popping up all over the board. Be it from decreased production times, to minimizing the amount of maintenance that is needed. Customer service and sales can also focus on increasing their product expertise so as to better serve customers. 
+
+### Pricing
+
+The market will pay whatever the market can support. The price of fruit obviously change depending on their country of origin, the season and weather conditions. Those are supply side changes, but what about demand? Various customers are prepared to purchase different items at drastically different prices. A taxi driver needs oil changes or car filters much more than a student who only drives home every other weekend.
```
Comment (filipstachura) [inline _posts_knitr/2017-07-20-improve-retail.Rmd]: Does it make sense to mention Uber here?

File: _posts_knitr/2017-07-20-improve-retail.Rmd [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+
+## Use Cases
+
+Retail and e-commerce both generate a surprisingly large amount of data in recent years. But without the proper tools in place, most of this data illustrates what you already know. I want to point out a few common uses of data science in retail. There are quite a few more that we could discuss, but not every company is ready for them. Contact us if you think you are ready for full automation!What I mean is that they are trying to skip steps that need to be done before we start completely automating processes. 
+
+### Product Recommendations
+
+Traditionally, retailers would have consultants in store that would use their own judgement to help customers find what they need. There are also encaps that present the highest bidder to customers without taking into consideration their interests or needs. Online, this would be based on the past revenue and would try and recommend the highest selling products. This didn’t really make any sense, as these products were doing quite well selling themselves and skewed the rest of the catalogue.
```
Comment (filipstachura) [inline _posts_knitr/2017-07-20-improve-retail.Rmd]: endcaps

File: _posts_knitr/2017-07-20-improve-retail.Rmd [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
...These customers are proven to be extremely tired and not have any desire to do their groceries in multiple places. As such, it was in Target’s interest to get these customers through the door for formula or diapers because they would then do their entire grocery shopping for the week. The life time value of these clients is very high, as there is low churn among them for a few years following pregnancy. But the hype was over glorified. The story itself revolves around a 16 year old girl, her father, a Target manager and a flier. The father had received fliers and coupons for maternal vitamins, diapers, etc. and came in to complain to the manager. The manager apologized and proceeded to call the father a few weeks later, but it turned out that the daughter was in fact pregnant. While the exact story never happened, target did find a way to predict pregnancies. It turns out that there are very specific items purchased when an individual is in their second trimester. Over 20 items taken into consideration were enough to give every customer a pregnancy chance score. This example illustrates the kinds of insights that can be discovered but only scratched the surface of what is possible.
```
Comment (filipstachura) [inline _posts_knitr/2017-07-20-improve-retail.Rmd]: add source to kdnuggets! `While the exact story never happened [^1]`

At the bottom:
```
[^1]: [jQuery](http://jquery.com/) is an old but still very popular library ...
```

File: _posts_knitr/2017-07-20-improve-retail.Rmd [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+
+## Summary
+
+We’ve looked at four examples of how data science and machine learning can be used in online and offline retail. The most important points I want to reiterate are the benefits of personalization and the true impact of trusting in data. It is wasteful to collect data and not take advantage of it. 
+
+[Get in touch with us](mailto:konrad.pabianczyk@appsilondatascience.com) if you want to see how we can help you!
```
Comment (filipstachura) [inline _posts_knitr/2017-07-20-improve-retail.Rmd]: How about? : Get in touch with us if you want to see in what ways your data can give you an advantage.

File: _posts_knitr/2017-07-20-improve-retail.Rmd [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+
+## Summary
+
+We’ve looked at four examples of how data science and machine learning can be used in online and offline retail. The most important points I want to reiterate are the benefits of personalization and the true impact of trusting in data. It is wasteful to collect data and not take advantage of it.
```
Comment (filipstachura) [inline _posts_knitr/2017-07-20-improve-retail.Rmd]: Idea:

```
... offline retail. While the number of possibilities is almost unlimited, the most important ...
```

File: _posts_knitr/2017-07-20-improve-retail.Rmd (hunk 4 above)
Comment (konradpab) [inline _posts_knitr/2017-07-20-improve-retail.Rmd]: Price surging has negative sentiment

**Pull request merged**